frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -78,3 +77,21 @@ export const UpsellCardComponent = styled.div<Variants>`
   border: 1px solid ${upsellColors.secondary};
   background-color: var(--mb-color-bg-white);
 `;
+
+export const UpsellBannerComponent = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 1rem;
+  overflow: hidden;
+  border: 1px solid var(--mb-color-border);
+  border-radius: 0.5rem;
+`;
+
+export const SecondaryCTALink = styled(ExternalLink)`
+  display: inline;
+  color: ${upsellColors.primary};
```

Comment:
Designs are slightly inconsistent. I went with a darker color as it provides better contrast.

For example:
[Upsell System Guidelines](https://www.notion.so/metabase/Upsell-System-Guidelines-72ce6f7402fb4c93a165e904dcf7a93c?pvs=4#73717148f12e4cf9b54f589427c8f1a0) suggests to use the darker color.

But the actual [Figma design] is showing the lighter and the brighter hue.
https://www.figma.com/design/kdZZogH2tWBVwnG03N7XXc/Upsell-System?node-id=262-21430&m=dev